lms/static/js/learner_dashboard/views/program_card_view.js

```diff
@@ -23,6 +25,31 @@
                 render: function() {
                     var templated = this.tpl(this.model.toJSON());
                     this.$el.html(templated);
+                    this.postRender();
+                },
+                postRender: function() {
+                    if(navigator.userAgent.indexOf('MSIE') !== -1 ||
+                        navigator.appVersion.indexOf('Trident/') > 0){
+                        /* Microsoft Internet Explorer detected in. */
+                        window.setTimeout( function() {
+                            this.reLoadBannerImage();
+                        }.bind(this), 100);
+                    }
+                },
+                // Defer loading the rest of the page to limit FOUC
+                reLoadBannerImage: function() {
+                    var $img = this.$('.program_card .banner-image'),
+                        imgSrcAttr = $img ? $img.attr('src') : {};
+                    
+                    if (!imgSrcAttr || imgSrcAttr.length < 0) {
+                        try{
+                            picturefill({
+                                reevaluate: true
+                            });
+                        }catch(err){
+                            //Swallow the error here
+                        }
+                    }
```

Comment:
I'm less concerned with the browser detection part; you could simply test the `reLoadBannerImage` function itself (spy on pictureFill and make sure it is called when there is no image src).    But honestly this is not a dealbreaker for me.